canonical-web-and-design/ubuntu.com
# Move tutorials JS to template to avoid race condition

## Done

Move tutorials JS to the template to fix page jump on load.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/tutorials
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click through to a tutorial
- Check that the page is scrolled to the top and doesn't jump


## Issue / Card

Fixes #6387
